ts/screens/authentication/IdpSelectionScreen.tsx

```diff
@@ -111,50 +115,62 @@ const styles = StyleSheet.create({
 /**
  * A screen where the user choose the SPID IPD to login with.
  */
-const IdpSelectionScreen: React.SFC<Props> = props => {
-  const goBack = () => props.navigation.goBack();
+class IdpSelectionScreen extends React.PureComponent<Props, {}> {
```

Comment:
why?

Reply:
Because there is no reason to recreate the same function at every render, it's a **bad** practice.

Reply:
yes, but should this change in the scope of this PR (_Show message when session is expired_)?

Reply:
you should ask yourself the following question: _can I accomplish the purpose of this PR without this change?_ 

if the answer is _yes_, then that change is out of scope

Reply:
to be clear, your suggestion is somewhat relevant (it's an optimization yes, but this component gets rendered very few times, so I would stick for less code for now), it's just that it shouldn't be part of this PR